pkg/asset/installconfig/powervs/client.go

```diff
@@ -746,3 +750,30 @@ func (c *Client) ServiceInstanceIDToCRN(ctx context.Context, id string) (string,
 
 	return "", nil
 }
+
+// GetDatacenterCapabilities retrieves the capabilities of the specified datacenter.
+func (c *Client) GetDatacenterCapabilities(ctx context.Context, region string) (map[string]bool, error) {
```

Comment:
you can do this change to avoid confusion

```suggestion
func (c *Client) GetDatacenterCapabilities(ctx context.Context, zone string) (map[string]bool, error) {
```